src/styles/blocks/card.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  display: flex;
+  width: 200px;
+  height: 408px;
+  box-sizing: border-box;
+  padding: 16px;
+  flex-direction: column;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: $white;
+
+  &__image {
+    margin-top: 16px;
+    align-self: center;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__header {
+    margin: 0;
+    margin-top: 40px;
+    color: $main-accent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  @mixin defaultTextPreset () {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__code-info {
+    margin: 0;
+    margin-top: 4px;
+    color: $secondary;
+    @include defaultTextPreset;
+  }
+
+  @mixin flexBetween {
+    display: flex;
+    justify-content: space-between;
+  }
```

Comment:
Move mixins and variables to separate files as you can use them in other places.